### PR TITLE
Initialize Form Validation

### PR DESCRIPTION
**Pull request by angeluray**
## Basic Check✔

- Created a new branch according to the Github Flow rules.
- Added form validation functions in JS file.
- Added a .css file that includes styles for Form Validation.
- Set up linters for HTML/CSS and JS making sure they work correctly.
- Added a .gitignore file to make sure any irrelevant files are ignored by git.
- Added changes doing pair programming according to the requirements.

## Project Requirements check ✔

- When the user submits the form, you check if the email is in lower case. - **Done.**
- If the validation is OK, the form is sent. - **Done.**
- If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent. - **Done.**
## Image Check ✔
photo_5107519244219820796_y
photo_5107519244219820797_y
*

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -143,15 +143,16 @@ <h2 class="contact-us" id="contact-me">CONTACT US</h2>
...
             built or a project that needs coding. I'd love to help with it </p>
-        <form action="https://formspree.io/f/xayvgowp" method="POST">
+        <form id="myForm" action="https://formspree.io/f/xayvgowp" method="POST">
             <ul class="my-form">
                 <li>
                     <label for="user-name">Full name</label>
                     <input type="text" name="user-name" id="user-name" placeholder="Full name" maxlength="30" required>
                 </li>
-                <li>
+                <li class="validatorContainer">
```
Comment (topeogunleye) [inline index.html:152]: - [ ] Great job adding your error message for "Emails written in Capital Letter", however pls as suggested by the previous reviewer, kindly add your error message close to your submit button and not above the textarea like you have done.

Screenshot 2022-07-23 021929

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @angeluray,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * Nothing to mention.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by angeluray** [comment]
## Visual check✔

- Fixed message position.

photo_5107519244219820806_y
photo_5107519244219820807_y

**Review by topeogunleye — APPROVED** [review]
### Hi @angeluray, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**